Rubrik/Public/New-RubrikDatabaseMount.ps1

```diff
@@ -0,0 +1,106 @@
+#Requires -Version 3
+function New-RubrikDatabaseMount
+{
+  <#  
+      .SYNOPSIS
+      Create a new Live Mount from a protected database
+      
+      .DESCRIPTION
+      The New-RubrikDatabaseMount cmdlet is used to create a Live Mount (clone) of a protected database and run it in an existing database environment.
+      
+      .NOTES
+      Written by Mike Fal for community usage from New-RubrikMount
+      Twitter: @Mike_Fal
+      GitHub: MikeFal
+      
+      .LINK
+      https://github.com/rubrikinc/PowerShell-Module
+
+      .EXAMPLE
+      New-RubrikDatabaseMount -id $db.id -targetInstanceId $db.instanceId -mountedDatabaseName 'BAR-LM' -recoveryDateTime (Get-date (Get-RubrikDatabase -id $db.id).latestRecoveryPoint)
+      Creates a new database mount named BAR on the same instance as the source database, using the most recent recovery time for the database. 
+      
+      $db=Get-RubrikDatabase -HostName FOO -Instance MSSQLSERVER -Database BAR
+
+  #>
+
+  [CmdletBinding(SupportsShouldProcess = $true,ConfirmImpact = 'High')]
+  Param(
+    # Rubrik id of the database
+    [Parameter(Mandatory = $true,ValueFromPipelineByPropertyName = $true)]
+    [String]$id,
+    # ID of Instance to use for the mount
+    [Alias('InstanceId')]
+    [String]$targetInstanceId,
```

Comment:
Please flip around the param case - user-facing params should be in PascalCase with an alias that uses the snake_case needed by the API